docker/openemr/7.0.4/openemr.sh

```diff
@@ -9,8 +9,7 @@
 # TODO: xdebug options should be given here
```

Comment:
some weirdness happened here in phpstorm with some cache thing that reverted prior changes. will fix these files.